tests/php/Security/MemberAuthenticator/SessionAuthenticationHandlerTest.php

```diff
@@ -49,12 +41,13 @@ public function testAuthenticateRequestStartsSessionWithSessionIdentifier()
 
         $session = new Session(null); // unstarted
         $session->set($handler->getSessionVariable(), $member->ID);
+        $session = new Session($session); // started
```

Comment:
`$session->start($req);` below was causing an issues calling `session_cache_limiter(null)` when running this unit-test in  non-islation mode, so I updated how the session is "started"